### DISCUSSION TITLE
Revolute joints ignore limits

### DISCUSSION THREAD

**Discussion by Teslatic** [bulletphysics/bullet3#3673] [General]
Follow up isse from #2831 (The changes suggested there have been applied so far)

**Problem description**
The tines of the grapple should be controllable in a fashion that they stop at their respective limits, specified in the according URDF file.

**Expected behaviour**
Tines close and stop at their limit. Likewise, the tines should open and stop when they are opened.

**Observed behaviour**
Tines rotate beyond their limits and continue rotation indefinitely. 

**How to recreate the behaviour**
python main_bullet_issue.py

Used versions
Python: 3.7.7
Pybullet: 2.7.8

Attached are the used files and a demo video.
[limit_issue.zip](https://github.com/bulletphysics/bullet3/files/4791254/limit_issue.zip)

Any help is appreciated.

**Comment by 1234shel** [comment]
Did you find a fix to this? I am also experiencing cases where the limits are violated.
